src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -407,6 +502,11 @@ internal ResultVersionOne CreateResult(Result v2Result)
                     }
                 }
 
+                if (v2Result.AnalysisTarget != null)
+                {
+                    result.Locations[0].AnalysisTarget = CreatePhysicalLocation(v2Result.AnalysisTarget);
```

Comment:
Set this property on every v1 `location` object.

Reply:
Done